types/rbush/index.d.mts

```diff
@@ -0,0 +1,184 @@
+export interface BBox {
```

Comment:
This entire file is very similar but slightly different to index.d.ts, is there an easy way to deduplicate it?

Reply:
Yes, you can actually just do `export * from "./index.js` or vice versa.